Add option to format microseconds and nanoseconds

mees- opened the pull request:
Also added an option to force ms to be formatted separately from seconds

Closes #24

mees- commented on index.js:
```diff
@@ -26,22 +26,42 @@ module.exports = (ms, options = {}) => {
 	};
 
 	const parsed = parseMs(ms);
+	// Manually add µs and ns to parsed, could be a pr to parse-ms
```
These lines could be removed if https://github.com/sindresorhus/parse-ms/pull/5 gets merged

FireController1847 commented:
but why

mees- commented:
@FireController1847 why what?

FireController1847 commented:
why would you need to pretty ms microseconds and nanoseconds

mees- commented:
because i want to show users times in a nice format and the times are less than a millisecond

FireController1847 commented:
I just don't see a use case for this... What does your program do that requires microseocond and millisecond precision?

mees- commented:
I'm working on a benchmarking thing, that has that kind of times

kolyaventuri commented on index.js:
```diff
@@ -32,16 +32,33 @@ module.exports = (ms, options = {}) => {
+	if (options.separateMs || ms < 1000) {
 		add(parsed.seconds, 'second', 's');
-		return '~' + ret[0];
+		if (options.formatSubMs) {
```
It works well, but it feels like the codebase starts to get a bit messy here? #28  would inherently fix this though, so I'm not sure that it's too big of a deal to address that immediately. I'm open to discussing that though, since resolving it now would deal with the tech debt.

sindresorhus commented on readme.md:
```diff
@@ -91,6 +96,20 @@ Default: `false`
+Type: `boolean`<br>
+Default: `false`
+
+Show milliseconds separately, this means they won't be included in the decimal part of the seconds
```
Can you include a short code example of before/after using this option with the output? Same with `formatSubMs`.

sindresorhus commented on readme.md:
```diff
@@ -32,6 +32,10 @@ prettyMs(1337, {compact: true});
 //=> '15 days 11 hours 1 minute 9 seconds'
 
+// formatSubMs option
+prettyMs(100.400080)
```
This needs to include the use of the option.

sindresorhus commented on test.js:
```diff
@@ -64,6 +64,17 @@ test('have a verbose option', t => {
+	t.is(m(0.400, {formatSubMs: true}), '400µs');
+	t.is(m(0.123571, {formatSubMs: true}), '123µs 571ns');
+	t.is(m(0.123456789, {formatSubMs: true}), '123µs 456ns');
+});
```
Can you add some tests that also shows `hours` and `seconds` units?

sindresorhus commented:
@mees- Can you fix the merge conflict?

mees- commented:
@sindresorhus I have added the test case and resolved the merge confict. I have also fixed an error that was caught by the addition of that test case.
